_posts/2021-08-24-devs-be-like.md

```diff
@@ -0,0 +1,70 @@
+
+Most employers want some kind of proof of a person's skills as a developer before making any decisions about hiring. Therefore, maintaining some projects or such are most probably an asset when thinking about switching jobs. Personally, I just can't find the motivation to do any of that. Maybe, if I wasn't this content at Solita and felt the need to find another job, I might actually do something about it.
+
+## Keeping your job
+
+No job in the world is static and stays the same. And software development is definitely not a field to change slowly. Being a professional developer you need to keep up with the new stuff. But that's the great part about this job! There is so much to learn and so many ways to develop your own skills further. 
+
+But that does not mean that you need to use your own free time to do all that learning. And obviously, if you want to make a more drastic move in your professional path, you might need to invest some of your own time to learn the basics of that new competence area. But I think that normal project work or time allocated by your company should provide you with enough chances to learn those new, hot and hip programming languages, frameworks, etc.
+
+And besides, software development is definitely not all about programming. Communication is super important. Good people skills come in very handy when you're part of a team. At least someone in the project should have pretty solid skills in coordination and organization. Each team needs a variety of skills and not every developer needs to be the best in programming. They can be a solid 8- (in Finland we are graded from 4 to 10, 10 being the best), but have good people skills. Or they can be a Jira-wizard and use some of their time organizing the team's backlog.
+
+At least here in Solita, learning new things on company time is very strongly recommended. Feedback is collected from us regularly and one of the things asked is whether or not we are able to learn and to develop ourselves in our daily work life. For me, this is a message that I do not have to spend my evenings learning new things. I can do that during the day and get paid for it. And that leaves me with more time after work to do things I enjoy and things that energize me.
+
+## Everything else
+
+Sure, it is possible that your life consists mostly of programming. But in the long run, it might be healthier to have some kind of balance. I really enjoy programming, but it is not even close to the top-3 most important things in my life. It might make it to top-10, maybe.
+
+Personally, variance in the things I do is super important. For a short while I might be able to focus on some free time programming tasks, but after a while, I know I'd just drop it and forget about it. 
```

Comment:
Well that's exactly what happens to hobby projects because it's fun to do POCs. You don't have to finish them like the production ready things you do at work.